doc/cprover-manual/properties.md

```diff
@@ -0,0 +1,295 @@
+## Automatically Generating Properties
+
+### What is a "Property"?
+
+We have mentioned *properties* several times so far, but we never
+explained *what* kind of properties CBMC can verify. We cover
+this topic in more detail in this section.
+
+CBMC uses
+[assertions](http://en.wikipedia.org/wiki/Assertion_(computing)) to
+specify program properties. Assertions are properties of the state of
+the program when the program reaches a particular program location.
+Assertions are often written by the programmer by means of the `assert`
+macro.
+
+In addition to the assertions written by the programmer, assertions for
+specific properties can also be generated automatically by CBMC, often
+relieving the programmer from writing "obvious" assertions.
```

Comment:
often relieving the programmer from expressing properties that should hold in any well-behaved program.